packages/amplication-design-system/package.json

```diff
@@ -55,17 +57,20 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "formik": "2.2.9",
     "prettier": "2.1.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-is": "16.13.1",
     "react-scripts": "5.0.0",
     "rollup": "2.63.0",
     "rollup-plugin-copy": "3.4.0",
     "rollup-plugin-postcss": "3.1.8",
     "rollup-plugin-typescript2": "0.32.1",
     "sass": "1.35.2",
-    "sass-loader": "12.6.0",
-    "style-loader": "3.3.1",
+    "sass-loader": "^12.6.0",
+    "style-loader": "^3.3.1",
+    "styled-components": "^5.3.3",
```

Comment:
Why is this not anchor? and why styled-components moved

Reply:
Same as above.